packages/modules-sdk/src/medusa-app.ts

```diff
@@ -59,10 +69,12 @@ export async function MedusaApp({
 }> {
   const modules: MedusaModuleConfig =
     modulesConfig ??
-    (await import(process.cwd() + (modulesConfigPath ?? "/modules-config")))
-      .default
-
-  const injectedDependencies: any = {}
+    (
```

Comment:
suggestion: This is not really webpack compatible due to the dynamic import composing, Should we allow to pass the path as an optionl input   and if not provided then do the dynamic loading? Like we did in the module loading mechanism where it either use a path or import it dynamically.

Reply:
We'll have to do it, just haven't implemented it in this PR. Will add support to pass the object reference, and we can chat about other alternatives we have.

Reply:
yes definetly, it is also not something we have to do now and can push it to later when we will add the differemt loading mechanism (config, packages or explicit references)